be/src/storage/tablet_updates.cpp

```diff
@@ -3582,10 +3567,10 @@ StatusOr<ExtraFileSize> TabletUpdates::_get_extra_file_size() const {
                 std::string filename = entry.path().filename().string();
 
                 if (filename.starts_with("index.l")) {
-                    ef_size.pindex_size += entry.file_size();
+                    _extra_file_size_cache.pindex_size.fetch_add(entry.file_size());
```

Comment:
should reset to 0 at the beginning of this function

Reply:
can we avoid traversing the files to get their sizes, maybe record the sizes directly in the metadata during ingestion and compaction?
traversing once may take a long time and affect the checkpoint of tablet meta.